purchase_stock_picking_invoice_link/models/stock_move.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2021 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
```

Comment:
```suggestion
# Copyright 2021 Tecnativa - Ernesto Tejeda
# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
```

Reply:
The license is correct. LGPL is only used in website modules.